plugins/modules/keycloak_userprofile.py

```diff
@@ -0,0 +1,727 @@
+                                        elements: str
+                                        default:
+                                          - admin
+                                          - user
+
+                            multivalued:
+                                description:
+                                    - Whether the attribute can have multiple values.
+                                type: bool
+                                default: false
+
+                            required:
+                                description:
+                                    - The roles that require this attribute.
+                                type: dict
+                                suboptions:
+                                    roles:
+                                        description:
+                                            - The roles for which this attribute is required.
+                                            - Supported values are V(admin) and V(user).
+                                        type: list
+                                        elements: str
+                                        default:
+                                          - user
+
+                    groups:
+                        description:
+                            - A list of attribute groups to be included in the User Profile.
+                        type: list
+                        elements: dict
+                        suboptions:
+                            name:
+                                description:
+                                    - The name of the group.
+                                type: str
+                                required: true
+
+                            display_header:
+                                description:
+                                    - The display header for the group.
+                                aliases:
+                                  - displayHeader
+                                type: str
+                                required: true
+
+                            display_description:
+                                description:
+                                    - The display description for the group.
+                                aliases:
+                                  - displayDescription
+                                type: str
+                                required: false
+
+                            annotations:
+                                description:
+                                    - The annotations included in the group.
+                                type: dict
+                                required: false
+
+                    unmanaged_attribute_policy:
+                        description:
+                            - Policy for unmanaged attributes.
+                            - Supported values are V(ENABLED), V(ADMIN_EDIT) and V(ADMIN_VIEW).
```

Comment:
These are already listed under `choices`, so there's no need to list them again.
```suggestion
```

Reply:
:+1: